src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -340,6 +349,7 @@ private void DisposeAndClearSubscriptions()
 
             _evaluationSubscriptionLinks.Clear();
```

Comment:
We only remove subscriptions when the project is closed.  What happens if one configuration is removed?  I think the subscription will hold many configuration related objects in the memory.  It is not a high priority for this release, but eventually we need address it too.